asm/internal/ll.bnf

```diff
@@ -504,6 +504,7 @@ NamedType
 Value
 	: LocalIdent
 	| Constant
+	| "undef"     << nil, nil >>
```

Comment:
Going forward we may need to introduce a notion for undefined values that is propagated through the AST and IR representations. For now, we can use `nil` simply to allow parsing of LLVM IR assembly files which contain `undef` values.